### DISCUSSION TITLE
Distributing GNS3 as a Flatpak or Appimage?

### DISCUSSION THREAD

**Discussion by saolof** [GNS3/gns3-gui#3470] [Ideas]
GNS3 has a lot of third party dependencies and is difficult to install on many linux distros including fedora, where you have to CMake key software. 

Would it be possible to distribute it in a flatpak/appimage/nix flake where it has access to key dependencies including say VPCS to make it run reliably on as many machines as possible?

**Comment by Knogle** [comment]
Any news on that?

**Reply by grossmj** [reply]
Someone started to package GNS3 with Flatpak: https://github.com/GNS3/gns3-gui/issues/2911
